tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -0,0 +1,208 @@
+# stdlib
+import operator
+
+# third party
+import pytest
+import torch
+
+from sympc.session import Session
+from sympc.tensor import MPCTensor
+
+
+def test_mpc_tensor_exception(clients) -> None:
+    alice_client, bob_client = clients
+    # TODO: for more than 2 parties
+    session = Session(parties=[alice_client, bob_client])
+
+    with pytest.raises(ValueError):
+        MPCTensor(secret=42, session=session)
+
+
+def test_reconstruct(clients) -> None:
+    alice_client, bob_client = clients
+    # TODO: for more than 2 parties
+    session = Session(parties=[alice_client, bob_client])
+    Session.setup_mpc(session)
+
+    x_secret = torch.Tensor([1, -2, 3.0907, -4.870])
+    x = MPCTensor(secret=x_secret, session=session)
+    x = x.reconstruct()
+
+    assert torch.allclose(x_secret, x)
+
+
+def test_sessions_without_setup_mpc(clients):
```

Comment:
NIT: I think we can remove this test, right? (we are testing the same behavior at line 12)

Reply:
done